lib/types/form-types.ts

```diff
@@ -145,4 +145,7 @@ export type FormRecord = {
 };
 
 // defines the fields for the form record that is available to unauthenticated users
-export type PublicFormRecord = TypeOmit<FormRecord, "name" | "deliveryOption" | "bearerToken">;
+export type PublicFormRecord = TypeOmit<
+  FormRecord,
+  "name" | "deliveryOption" | "bearerToken" | "email"
```

Comment:
`email` does not exist on FormRecord and is a part of `deliveryOption` which is already ommited.

Reply:
Removed  `email`